modules/checkout/commerce_checkout.module

```diff
@@ -34,6 +34,12 @@ function commerce_checkout_theme() {
     'commerce_checkout_form__with_sidebar' => [
       'base hook' => 'commerce_checkout_form',
     ],
+    'commerce_checkout_order_summary' => [
+      'variables' => [
+        'order_entity' => NULL,
+        'checkout_flow_step' => '',
```

Comment:
We don't mention a "checkout flow step" anywhere else in the codebase. It's a checkout step, so let's use "checkout_step"